src/lints/inherent_method_must_use_added.ron

```diff
@@ -1,7 +1,7 @@
 SemverQuery(
     id: "inherent_method_must_use_added",
     human_readable_name: "inherent method #[must_use] added",
-    description: "An inherent method has been marked with #[must_use].",
+    description: "A method or associated fn has been marked with #[must_use].",
```

Comment:
```suggestion
    description: "An inherent method or associated fn has been marked #[must_use].",
```
"Inherent" is a load bearing word here, it means the `impl` was on the type itself and not on a trait implemented for that type. This lint only looks at associated functions (including methods) of inherent impls.